modules/dnn/src/layers/elementwise_layers.cpp

```diff
@@ -355,7 +355,7 @@ struct ReLUFunctor : public BaseFunctor
                backendId == DNN_BACKEND_CANN;
     }
 
-    void apply(const float* srcptr, float* dstptr, int len, size_t planeSize, int cn0, int cn1) const
+    void apply(const float* srcptr, float* dstptr, int, int len, size_t planeSize, int cn0, int cn1) const
```

Comment:
I propose to restore new parameter name and use `CV_UNUSED()` for it. It makes code cleaner for readers.